SpatialGDK/Documentation/index.md

```diff
@@ -6,79 +6,6 @@ Welcome to the SpatialOS and Unreal GDK community.
 
 Our SpatialOS community shapes the functionality of the Unreal GDK, so make sure you are part of the conversation. Come and find us on [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) or talk to us on the **Feedback** section in our [forums](https://forums.improbable.io/) (use the **unreal-gdk** tag - [this link](https://forums.improbable.io/new-topic?category=Feedback&tags=unreal-gdk) takes you there and pre-fills the category and tag). You can also [sign up for SpatialOS Unreal GDK updates](http://go.pardot.com/l/169082/2018-06-15/27ld2t) direct to your mailbox.
```

Comment:
Ernie is working on the welcome page / index page for this - I will incorporate this first pass into the GDoc.
I think, like Unity, we can use the same text for the welcome/index page for the initial release and then review this? Does this work OK?
I have added your edits to the Google doc here, so Ernie knows what we are working from: https://docs.google.com/document/d/16pmz7wNgZzE2VPhDwGxc5pyXtgt-h-DPlEqCGsRKvXI/edit#